### PR TITLE
{bio}[gfbf/2023a] Seaborn v0.12.2

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 8 out of 8 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/f451c0a96f03ce0727f4e61e7b56e559 for a full test report.

File: easybuild/easyconfigs/s/Seaborn/Seaborn-0.12.2-gfbf-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
+
+name = 'Seaborn'
+version = '0.12.2'
```
Comment (verdurin) [inline easybuild/easyconfigs/s/Seaborn/Seaborn-0.12.2-gfbf-2023a.eb:4]: This looks fine, but `0.13.0` is out?

**Comment by boegel** [comment]
superseded by #19772

**Pull request closed without merging**